### DISCUSSION TITLE
Added LM Studio Support to Config File/llm_wrapper

### DISCUSSION THREAD

**Discussion by JonathanMitchell1234** [TheBlewish/Automated-AI-Web-Researcher-Ollama#27] [Ideas]
@TheBlewish I added some code to the config file and the llm_wrapper to support LM Studio generated endpoints, which is working (I already had LM studio installed and I didn't feel like redownloading Ollama lol so I figured I'd just make modifications) If you're interested in supporting it with ollama, let me know and I'll send a pull request.

**Comment by synth-mania** [comment]
I think PR #1 already has that covered, and so does another that is still open. No need to reinvent the wheel, clearly there is a lot of interest in using a general purpose openAI compatible endpoint config. Hopefully we can settle on a solution soon

**Reply by JonathanMitchell1234** [reply]
Right on! I didn't notice that they said OpenAI-like endpoints in that when I was skimming it earlier 👍.

**Reply by TheBlewish** [reply]
I am working on testing the stuff in PR 1 myself, as soon as I see it work i'll merge it, sorry for the delay everyone!
Rest assured I ain't trying to limit people's options I just didn't realize what people meant by the OpenAI endpoint stuff initially.

**Reply by TheBlewish** [reply]
It has been merged! Let me know yall run into any issues, but in my test it worked!
